fifthHW/first_task.py

```diff
@@ -0,0 +1,17 @@
+#1 task
+import os
+import random
+
+folder = "./test/data"
+os.makedirs(folder, exist_ok=True)
+data = []
+for i in range(100):
+    left_operand = random.randint(1, 10)
+    right_operand = random.randint(1, 10)
+    operation = random.randint(1, 3)
+    data.append((left_operand, right_operand, operation))
+    filename = "./test/data/data.txt"
+    with open(filename, "w") as file:
```

Comment:
you open your file and rewrite it in every iteration. it is better to put in outside loop